app/views/conferences/edit_notifications.html.haml

```diff
@@ -17,7 +17,7 @@
   %h1 Edit
 
   .row
-    .span6.offset8.label
+    .span6.offset8.label{ :style => "text-transform: none !important" }
```

Comment:
So I noticed that the .label class was only used for background color. I added a .info-block css style and used that instead of the bootstrap .label class.